src/component/list.js

```diff
@@ -50,6 +50,7 @@ export class List extends Component {
         ref={component => (this.list = component)}
         dataSource={this.dataSource}
         renderHeader={this.props.renderHeader}
+        stickyHeaderIndices={[0]}
```

Comment:
Since we're setting this globally for all `List` instances, we'll also need to add `style={{ backgroundColor: color.white }}` to the `ListHeader` in list-story in storybook.

Reply:
Done.